pypose/module/dynamics.py

```diff
@@ -0,0 +1,196 @@
+
+class LTI(_System):
+    r'''
+    A sub-class of '_System' to represent Linear Time-Invariant system.
+    
+    Args:
+        A, B, C, D (:obj:`Tensor`): The coefficient matrix in the state-space equation of LTI system
+        c1, c2 (:obj:`Tensor`): Bias generated by system
+        
+    Note:
+        According to the actual physical meaning, the dimensions of A, B, C, D must be the same,
+        whether in the batch case or not.
+        
+        The system is time invariant.
+    '''
+    def __init__(self, A, B, C, D, c1=None, c2=None):
+        super(LTI, self).__init__(time=False)
+        assert A.ndim == B.ndim == C.ndim == D.ndim, "Invalid System Matrices dimensions"
+        self._A, self._B, self._C, self._D = A, B, C, D
+        self._c1, self._c2 = c1, c2
+
+    @property
+    def c1(self):
+        return self._c1
+    
+    @property
+    def c2(self):
+        return self._c2
+    
+    def forward(self, x, u):
+        r'''
+        Args:
+            x (:obj:`Tensor`): The state of LTI system,
+            u (:obj:`Tensor`): The input of LTI system.
+
+        Return:
+            Tensor: The solved batched tensor, which means state-transition and observation of the system.
+
+            
+        Every linear time-invariant lumped system can be described by a set of equations of the form
+        which is called the state-space equation.
+        
+        .. math::
+            \begin{align*}
+                z_{i} = A_{i} \times x_{i} + B_{i} \times u_{i} + c_1 \\
+                y_{i} = C_{i} \times x_{i} + D_{i} \times u_{i} + c_2 \\
+            \end{align*}
+            
+        where :math:`x` is actually :math:`\dot{x}`, the differential form of :math:`x`.
+        
+        Let the input be matrix :math:`A`, :math:`B`, :math:`C`, :math:`D`, :math:`x`, :math:`u`, :math:`c_1`, :math:`c_2`. 
+        :math:`x_i` represents each individual matrix in the batch. 
+        
+        Note:
+            -:math:`x`, :math:`u` could be single input or multiple inputs
+
+            -:math:`A`, :math:`B`, :math:`C`, :math:`D` can only be two-dimensional matrices or the batch
+             In the batch case, their dimensions must be the same as those of :math:`x`, :math:`u`. 
+             :math:`A`, :math:`B`, :math:`C`, :math:`D` and :math:`x`, :math:`u` are multiplied separately for each channel.
+             
+            -For a System with p inputs, q outputs, and n state variables, :math:`A`, :math:`B`, :math:`C`, :math:`D` are n*n n*p q*n and q*p constant matrices.
+             
+            -Note that variables are entered as row vectors.
+
+        Example:
+            >>> A = torch.randn((2,3,3))
+                B = torch.randn((2,3,2))
+                C = torch.randn((2,3,3))
+                D = torch.randn((2,3,2))
+                c1 = torch.randn((2,1,3))
+                c2 = torch.randn((2,1,3))
+                x = torch.randn((2,1,3))
+                u = torch.randn((2,1,2))
+            >>> A
```

Comment:
In this example, it is not necessary to print out A, B, C, D, c1, c2, x, u explicitly. User can do them if they want. Let's just keep the randn part and the final lti part.